### DISCUSSION TITLE
LingmoOS 路线图讨论

### DISCUSSION THREAD

**Discussion by elysia-best** [LingmoOS/Community#2] [社区通知]
@Intro-iu @chun-awa 有关路线图的讨论和细分可以在这个帖子进行。本帖请用中文交流。

**Comment by elysia-best** [comment]
大家有建议的话，可以填写本次的调查问卷哦~
https://forms.office.com/e/0MUf5iBQQq

**Comment by devLythen** [comment]
余以为当下宜募集二三贤士，或斧之，或重构之，共议KDE6,Qt6及wayland之更替。

**Reply by elysia-best** [reply]
然也，仁兄
